app/Models/Client.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+use Laravel\Passport\Client as PassportClient;
+
+class Client extends PassportClient
```

Comment:
Should this have a more specific name like `OAuth2Client` or `PassportClient`?

Reply:
good point, I agree. I'll rename it tomorrow night

Reply:
Renamed to `OAuth2Client` to match other places